Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -176,7 +176,13 @@ public let CodeGenerators: [CodeGenerator] = [
         } else {
             propertyName = b.genPropertyNameForWrite()
         }
-        let value = b.randVar()
+        var propertyType = b.type(ofProperty: propertyName)
```

Comment:
I think we need the same logic in `CreateObject`, since that also defines properties